static/bidder-info/dxkulture.yaml

```diff
@@ -0,0 +1,16 @@
+endpoint: "https://ads.kulture.media/pbs"
+maintainer:
+  email: "devops@kulture.media"
```

Comment:
@kmdevops We have sent a verification mail to devops@kulture.media. Please reply back with "Received" message to complete the verification process.
Thank you!

Reply:
@Sonali-More-Xandr  Responded, thank you